onlineteamscan_v2_fe/pages/teamdetail/_teamdetail.vue

```diff
@@ -145,29 +149,35 @@ export default {
       ],
       nameRules: [
         value => !!value || 'Vereist',
-        v => /^[a-zA-Z]+$/.test(v) || 'Moet geldig zijn',
+        v => /^[^-\s][ áàíóúéëöüñÄĞİŞȘØøğışÐÝÞðýþa-zA-Z_\s-]*$/.test(v) || 'Moet geldig zijn',
         v => v.length <= 70 || 'Max 70 characters'
       ],
     }
   },
   created() {
-    this.$axios.get(`teammembers/team/${1}`).then(res => this.teamMembers = res.data).catch(err => console.log(err))
+    this.$axios.get(`teams/members/${this.$route.params.teamdetail}`).then(res => this.team = res.data).catch(err => console.log(err))
   },
   computed: {
     getActiveTeamMembers() {
-      return this.teamMembers.filter(x => x.isActive === true)
+      return this.team.teamMembers?.filter(x => x.isActive === true)
```

Comment:
Je returned hier nu mogelijk undefined, niet? Als team.teamMember? undefined returned, krijg je volgens mij geen lege array.

Het lijkt me beter om getActiveTeamMembers een lege array te laten terugsturen indien niets gevonden wordt.

Reply:
idem voor andere functies in deze file.